symphony-bdk-examples/bdk-core-examples/src/main/java/com/symphony/bdk/examples/UserExampleMain.java

```diff
@@ -31,5 +35,10 @@ public static void main(String[] args) throws Exception {
     log.info("Account type: {}", userDetailList.get(0).getUserAttributes().getAccountType());
     log.info("Company: {}", userDetailList.get(0).getUserAttributes().getCompanyName());
     log.info("Role: {}", userDetailList.get(0).getRoles());
+
+    AuthSession oboSession = bdk.obo("hong.le");
+    StreamFilter streamFilter = new StreamFilter().addStreamTypesItem(new StreamType().type(StreamType.TypeEnum.IM));
+    List<StreamAttributes> streamsList = bdk.streams().listStreams(oboSession, streamFilter);
```

Comment:
According to https://perzoinc.atlassian.net/browse/APP-2908, we wanted to have something like :
`bdk.messages(Obo.username("user.name")).send(STREAM, MESSAGE);` or `bdk.streams(Obo.username("user.name")).listStreams(streamFilter);`
Why changing it to `bdk.streams().listStreams(oboSession, streamFilter)` ?

Reply:
Because we don't want to initialize a new service instance each time we pass an Obo to `bdk#messages`, it will cause an memory issue if we call it a lot of times. (For ex: if we want to call an api OBO for 1000 thousands user, we have to instantiate 1000 message service instances)